### DISCUSSION TITLE
mkdocs support

### DISCUSSION THREAD

**Discussion by avelino** [tinacms/tinacms#3802] [Ideas]
Hey Tina folks. I'm curious to know where we are with [mkdocs](https://www.mkdocs.org/) support. Thanks!

**Comment by scottgallant** [comment]
👋 Hi @avelino, we haven't done much testing with mkdocs, to my knowledge.  Have you tried this yet? 

My sense is that this should work if you're using [basic editing](https://tina.io/docs/contextual-editing/overview/) but not with contextual editing (so, no live preview).  

Is your repo public? If so, can you share it so we can take a look?

**Reply by berot3** [reply]
Im curious too. Did anything happen in this regard? I wish I had the time to do a short test… maybe later 😄👍🏻
